subworkflows/local/utils_nfcore_detaxizer_pipeline/main.nf

```diff
@@ -186,10 +191,11 @@ def toolCitationText() {
     def citation_text = [
             "Tools used in the workflow included:",
             "FastQC (Andrews 2010),",
-            "fastp (Chen et al. 2018)",
-            "Kraken2 (Wood et al. 2019),",
-            !params["skip_blastn"] ? "BLAST (Altschul et al. 1990)," : "",
-            !params["skip_blastn"] | params["enable_filter"] ? "seqkit (Shen et al. 2016)," : "",
+            params["preprocessing"] ? "fastp (Chen et al. 2018),": "",
+            params["classification_kraken2"] | !params["classification_bbduk"] & !params["classification_kraken2"] ? "Kraken2 (Wood et al. 2019)," : "",
```

Comment:
I don't get the second `classification_kraken2` here? If you give neither you put in Kraken2 anyway?

Reply:
Edit: Ok looking at the code indeed that is the case :sweat_smile: